src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getMetadataFromStatement.test.ts

```diff
@@ -0,0 +1,27 @@
+import * as assert from 'assert';
+
+import getMetadataFromStatement
+  from '../../../../service/storeStatements/queriables/getMetadataFromStatement';
+import { sequencingInteractionActivityStatement, statementDefaults } from './fixtures/statements.fixture';
+
+describe('Retrieve metadata from statement', () => {
+  it('should retrieve metadata from statement', () => {
```

Comment:
We seem to be missing a test case for the case where a duration is provided and a sequence is provided.